Riot/Modules/MatrixKit/Utils/EventFormatter/MXKEventFormatter.m

```diff
@@ -1364,6 +1364,18 @@ - (NSAttributedString *)attributedStringFromEvent:(MXEvent*)event
                             body = [VectorL10n noticeInvalidAttachment];
                             *error = MXKEventFormatterErrorUnsupported;
                         }
+                        else
```

Comment:
Now there's an `else`, the file message block might be a little bit easier to read if this was flipped around and started with `if ([self isSupportedAttachment:event])` and the `body = body ? body : [Vector…` was part of that happy path.

Reply:
Good catch!